src/Components/Samples/BlazorUnitedApp/Program.cs

```diff
@@ -26,6 +27,6 @@
 
 app.UseRouting();
 
-app.MapRazorComponents();
+app.MapRazorComponents<MainLayout>();
```

Comment:
This might be replaced in the future with `app.MapRazorComponents<App>` if we end up having a root component for Blazor united as we do for Webassembly and Server.

Reply:
As it stands, the type `T` here is just a way to indicate the containing assembly, right? It's unfortunate that C# does not have an `assemblyof(MyApp)` (where `MyApp` is not a type, but rather is one of the assemblies in the compilation) keyword but I agree this would make sense if `T` goes on to serve the role of a root component.